test/configCases/plugins/define-plugin/webpack.config.js

```diff
@@ -33,7 +33,8 @@ module.exports = {
 					CODE: "(1+2)",
 					REGEXP: /abc/i,
 					STRING: JSON.stringify("string")
-				}
+				},
+				SUB1: { a: 1 }
 			},
```

Comment:
I wonder if that also works:

```suggestion
			},
			OBJECT.SUB2: { a: 1 },
```

Reply:
fixed